tools/kinatest_y_shaker/Kinamine_PS.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="kinamine_shaker" name="Kinamine_for_Peptide_Shaker" version="0.3.0">

```

Comment:
the name is exposed to the user and should not container a `_` it should read like a sentence combined with the description.